db/migrate/20150923142005_add_column_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToItems < ActiveRecord::Migration
+  def change
+    add_column :items, :category_id, :integer
```

Comment:
This should be null: false since your link in your views depends on the items category.